docker-compose.yml

```diff
@@ -32,6 +32,8 @@ services:
     environment:
       PGUSER: postgres
       PGPASSWORD: postgres
+    volumes:
+      - ./scripts:/scripts
```

Comment:
this allows us to extend test tables a little more flexible than downloading a dump, wdyt?

Reply:
I like it, but how do keep these scripts updated?

Reply:
That's the good part. Other than the dump file we're currently using, we can actually update and extend those sql without uploading another dump to the repository.

But, now that I look at it again, this name is confusing. It should be `testdata` (is the convention isnt it?).

Reply:
fixed